### PR TITLE
Style changes + simplification

### PR DESCRIPTION
**Pull request by diogoscf**

### CODE REVIEW

File: src/core/core.ts [hunk 1]
```diff
@@ -25,8 +25,8 @@ export class Core {
   /**
    * Creates document for this group in Cloud Firestore.
    */
-  async initNewGroup(groupId): Promise<void> {
-    await admin.firestore().collection("groups").doc(groupId).create({
+  initNewGroup(groupId): void {
```
Comment (bartekpacia) [inline src/core/core.ts:28]: It performs an async operation, so let's make it clear. I'd leave this as it was before.

File: src/telegram/bot_telegram.ts [hunk 2]
```diff
@@ -77,18 +77,18 @@ export class EnforcingTelegramBot extends TelegramBot {
     // Perform some initial setup when added to a group
     this.on("new_chat_members", async (msg) => {
       msg.new_chat_members?.forEach(async (user) => {
-        // TODO Find a better way, don't hardcode username
-        if (user.username === "LangPolizeiBartekBot") {
+        const botData = await this.getMe()
```
Comment (bartekpacia) [inline src/telegram/bot_telegram.ts:80]: plz check

File: src/core/core.ts (hunk 1 above)
Comment (diogoscf) [reply src/core/core.ts:28]: a bit useless but ok

File: src/telegram/bot_telegram.ts (hunk 2 above)
Comment (diogoscf) [reply src/telegram/bot_telegram.ts:80]: will do

Comment (diogoscf) [reply src/telegram/bot_telegram.ts:80]: will do

**Comment by diogoscf** [comment]
btw the test failing isn't my fault

File: src/telegram/bot_telegram.ts (hunk 2 above)
Comment (diogoscf) [reply src/telegram/bot_telegram.ts:80]: Tested, it works

File: src/core/core.ts (hunk 1 above)
Comment (diogoscf) [reply src/core/core.ts:28]: done

Comment (bartekpacia) [reply src/core/core.ts:28]: not useless
if you don't want to wait, you can just do `initNewGroup(groupId)` without awaiting it
and it's more explicit, doesn't return until it's done

**Comment by bartekpacia** [comment]
yeah
`PROJECT_ID` is missing, hmm
will fix after English and PhysX

**Pull request merged**